DependencyInjection/Configuration.php

```diff
@@ -167,4 +168,27 @@ private function getOrganizeMigrationsModes() : array
 
         return $namesArray;
     }
+
+    /**
+     * Returns the correct deprecation param's as an array for setDeprecated.
+     *
+     * Symfony/Config v5.1 introduces a deprecation notice when calling
+     * setDeprecation() with less than 3 args and the getDeprecation method was
+     * introduced at the same time. By checking if getDeprecation() exists,
+     * we can determine the correct param count to use when calling setDeprecated.
```

Comment:
So is it `setDeprecation` or `setDeprecated`? Also, please add the missing parentheses